lib/src/feature/authentication/widgets/login_form.dart

```diff
@@ -5,9 +5,9 @@ import 'package:exerlog/src/widgets/text_field.dart';
 import 'package:flutter/material.dart';
 
 class LoginForm extends StatelessWidget {
-  const LoginForm(this.controller);
+  const LoginForm(this._controller);
 
-  final AuthenticationController controller;

```

Comment:
I'm assuming this wasn't a private field before because if you would make this a named construction now, it would give an error because underscores aren't allowed in named constructors.

Reply:
And seeing that it's a final variable, in this case it's not really a big deal it wasn't a private variable, but it's not bad that you change it 👍.

Reply:
After reading this and the edit from @PickleNik I have removed the leading underscore as it seems like an unnecessary change to make to the code base. Thank you both for explaining